dev/components/Login.jsx

```diff
@@ -1,10 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {Navbar, NavItem, Nav} from 'react-bootstrap'
+import { Button, Dropdown } from 'semantic-ui-react'
```

Comment:
Let's try to choose and stick with only one UI library. Bootstrap or semantic, pick one please.